src/Parser/Parser.php

```diff
@@ -21,11 +22,15 @@ class Parser
     /** @var MetadataDriver $metadata */
     protected $metadata;
 
-    public function __construct(Config $config, ResourceManager $resourceManager, MetadataDriver $metadata)
+    /** @var Users $users */
+    protected $users;
+
+    public function __construct(Config $config, ResourceManager $resourceManager, MetadataDriver $metadata, Users $users)
```

Comment:
If BC is being observed, this is a break.

Reply:
Oh! Good point!

Thanks for checking @GawainLynch  👍